src/s3/builders/put_object.rs

```diff
@@ -52,6 +52,7 @@ pub struct CreateMultipartUpload {
     tags: Option<HashMap<String, String>>,
     retention: Option<Retention>,
     legal_hold: bool,
+    content_type: Option<String>,
```

Comment:
We need to add method to be able to set this parameter in the CreateMultipartUpload struct as the struct members are not public

Reply:
done